setup.py

```diff
@@ -20,5 +20,7 @@
     url='https://github.com/AlgoETS/BullETS',
     package_dir={'': 'src'},
     packages=find_packages(where='src'),
-    install_requires=requirements
+    install_requires=requirements,
+    package_data={'': ['requirements.txt']},
+    include_package_data=True
```

Comment:
We can remove this line now since we have `package_data`

Reply:
No, from what I've seen online we need both (https://github.com/aws-quickstart/taskcat/issues/143)

Reply:
It doesn't look necessary according to setuptools' documentation: 
https://setuptools.readthedocs.io/en/latest/userguide/datafiles.html

Reply:
fixed, you are right it does not seem necessary